File: 8-kyu/grader/python/solution.py
def grader(score):
    score = round(score * 100)
    if not (60 <= score <= 100):
        return 'F'
    if score < 70:
        return 'D'
    elif score < 80:
        return 'C'
    elif score < 90:
        return 'B'
    return 'A'

File: 8-kyu/grader/javascript/solution.js
const grader = score => {
  if (score < 0.6 || score > 1.0) {
    return "F";
  } else if (score < 0.7) {
    return "D";
  } else if (score < 0.8) {
    return "C";
  } else if (score < 0.9) {
    return "B";
  }
  return "A";
};
